server/region_handler_test.go

```diff
@@ -282,6 +300,46 @@ func (ts *TidbRegionHandlerTestSuite) TestGetMvcc(c *C) {
 	err = decoder.Decode(&data2)
 	c.Assert(err, IsNil)
 	c.Assert(data2, DeepEquals, data)
+
+	// tests for normal index key
+	resp, err = http.Get("http://127.0.0.1:10090/mvcc/index/tidb/test/idx1/1?a=1&b=2")
+	c.Assert(err, IsNil)
+	decodeKeyMvcc(resp.Body, c, true)
+
+	resp, err = http.Get("http://127.0.0.1:10090/mvcc/index/tidb/test/idx2/1?a=1&b=2")
+	c.Assert(err, IsNil)
+	decodeKeyMvcc(resp.Body, c, true)
+
+	// tests for index key which includes null
```

Comment:
Any tests for index key that contains an empty string?

Reply:
Done.